config/testgrids/conformance/conformance-all.yaml

```
@@ -306,9 +306,9 @@ test_groups:
 
 #ibm ppc64le test results
 - name: ppc64le-conformance
-  gcs_prefix: ppc64le-kubernetes/logs/ci-conformance-kubernetes
+  gcs_prefix: ppc64le-kubernetes/logs/periodic-kubernetes-conformance-test-ppc64le
 - name: ppc64le-unit
-  gcs_prefix: ppc64le-kubernetes/logs/ci-unit-kubernetes
+  gcs_prefix: ppc64le-kubernetes/logs/periodics-kubernetes-bazel-test-ppc64le
```

Is the extra `s` a typo?
```suggestion
  gcs_prefix: ppc64le-kubernetes/logs/periodic-kubernetes-bazel-test-ppc64le
```

---

I think we have a typo in our job names, @Rajalakshmi-Girish let's fix them in our repos and make changes here as well.